tests/test_domain_c.py

```diff
@@ -591,6 +591,33 @@ def test_build_domain_c_anon_dup_decl(app, status, warning):
     assert "WARNING: c:identifier reference target not found: @b" in ws[1]
 
 
+@pytest.mark.sphinx(testroot='domain-c', confoverrides={
+    'nitpicky': True,
+    'nitpick_ignore': [
```

Comment:
Thank you for adding testcases. LGTM. But it would be better if they will move to `tests/test_config.py` because `nitpicky_warning` is a common feature, not only for C/C++. Could you move them, please?

Reply:
Do you want me to just move the tests to a different file or to also create a separate `testroot` instead of reusing `domain-c` and `domain-cpp`? I haven't looked into how the whole test root setup works yet.